src/main/java/hadouken/SimpleMessage.java

```diff
@@ -56,6 +57,7 @@ public JsonNode getJsonBody() throws IOException {
   }
 
   public org.codehaus.jackson.JsonNode getXmlBody() throws IOException {
+    // NOTE(justin.morgan): Why do we have two implementations of jackson.JsonNode? o_O
```

Comment:
Because this code was copied from the main repo here and it was not removed. = p